<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelPic">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>開啟影像</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderImageBrightness">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelBrightAdjustValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>61</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderImageBrightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>brightAdjust()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openImage()</slot>
  <slot>brightAdjust()</slot>
 </slots>
</ui>
